libraries.cmake/kissfft.cmake

```diff
@@ -0,0 +1,185 @@
+##################################################
+###       KISSFFT   														 ###
+##################################################
+
+MACRO( OPENMS_CONTRIB_BUILD_KISSFFT )
+  OPENMS_LOGHEADER_LIBRARY("kissfft")
+
+  if(MSVC)
+    set(ZIP_ARGS "x -y -osrc")
+  else()
+    set(ZIP_ARGS "xzf")
+  endif()
+  OPENMS_SMARTEXTRACT(ZIP_ARGS ARCHIVE_KISSFFT "kissfft" "README")
+  
+  #patch
+  set(PATCH_FILE "${PROJECT_SOURCE_DIR}/patches/kissfft/kiss_fftnd.h.patch")
```

Comment:
Great observation will do that... I thought it used relative references in header files on the tree structure, but I did.